.github/workflows/avail-light-bootstrap.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         include:
-          - os: ubuntu-20.04
+          - os: ubuntu-22.04
```

Comment:
Any concerns to bump it to 24.04?

Reply:
Fails to run on hex due to the version of glibc (requires 2.38, and available is 2.35).